include/bluetooth/direction.h

```diff
@@ -101,6 +101,8 @@ struct bt_df_per_adv_sync_iq_samples_report {
 	uint8_t slot_durations;
 	/** Status of received PDU with CTE (@ref bt_df_packet_status). */
 	uint8_t packet_status;
+	/** Value of the paEventCounter of the AUX_SYNC_IND PDU. */
+	uint16_t per_evt_counter;
```

Comment:
Unrelated to this PR, but someone should look into the ordering of struct members here. It seems like there might be unnecessary padding because of it.

Reply:
I'll take a look on this. Thanks @jhedberg